packages/amplication-data-service-generator/src/util/prepare-yaml-file.ts

```diff
@@ -6,8 +6,7 @@ export function prepareYamlFile(
   updateProperties: { [key: string]: any }[]
 ): string {
   const parsed = YAML.parse(yamlFileContent);
-  updateProperties &&
-    updateProperties.forEach((updateProperty) => merge(parsed, updateProperty));
+  updateProperties && merge(parsed, ...updateProperties);
```

Comment:
we need to make sure it's not nested objects cause merge is flat merge and not deep merge. maybe it's better to use deep merge then merge

Reply:
I should revert it. it was just for test